ExtLibs/px4uploader/Uploader.cs

```diff
@@ -500,6 +500,8 @@ public bool __trySync()
             byte c = __recv()[0];
             if (c != (byte)Code.INSYNC)
                 return false;
+
+            c = __recv()[0];

```

Comment:
Not sure about this change, why the extra read?

Reply:
Look at the code for sync() - it is absolutely required. This function clealy never worked.

Reply:
indeed, a 7 year old bug.....

Reply:
Hah, yes - I only found it by comparing with the python uploader